### DISCUSSION TITLE
How to configure LDAP on Apache Superset

### DISCUSSION THREAD

**Discussion by AhmedApache** [apache/superset#29310] [Q&A / Help]
I am trying to configure LDAP to enable my organization set up User Group on Apache Superset, I can't find a way around it.
Has any one done this before or know how to go about the configuration?

**Comment by scaffarelli** [comment]
I'm trying to find a way and I'm not able to get it working, was you able ?

**Comment by lodu** [comment]
You can use this: https://flask-appbuilder.readthedocs.io/en/latest/security.html
